### ISSUE TITLE
Issue with Private Channels

### ISSUE THREAD

**Issue by AmeerHafez** [nRF24/RF24Audio#44]
So i have set up two communicating transceivers using broadcast. but when I set up another pair of transceivers there is so much interference :( !.. how to avoid this?

**Comment by TMRh20** [comment]
You would have to change the radio channel on one set of devices. Call radio.setChannel(value 1-126); after calling rfAudio.begin();

**Comment by TMRh20** [comment]
The broadcast feature allows you to transmit to individual radio numbers or groups of radios sharing the same number. It’s all using multicast so switching radio channel is not required unless running multiple sets of devices.

**Comment by AmeerHafez** [comment]
ahhh ok.. So if i change channels with radio.setChannel for multiple devices I don't need to use broadcast I guess?

**Comment by TMRh20** [comment]
Yeah, broadcast is for when using a single transmitter to multiple devices
